### PR TITLE
Post submission submitted application

### PR DESCRIPTION
**Pull request by dreamfall**

### CODE REVIEW

File: app/decorators/form_answer_decorator.rb [hunk 1]
```diff
@@ -107,6 +107,10 @@ def sic_code_name
     end
   end
 
+  def application_in_progress?
```
Comment (jaleszek) [inline app/decorators/form_answer_decorator.rb]: there is another state `application_in_progress` I hope it won't be misleading

Comment (dreamfall) [inline app/decorators/form_answer_decorator.rb]: @jaleszek you're right, I should change this method's name

File: app/views/content_only/post_submission/_submitted.html.slim [hunk 2]
```diff
@@ -0,0 +1,19 @@
+- applications = FormAnswerDecorator.decorate_collection(award_applications.where(state: %w(assessment_in_progress submitted not_submitted application_in_progress)))
```
Comment (jaleszek) [inline app/views/content_only/post_submission/_submitted.html.slim]: Technically we shouldn't have to look for a `submitted` and `application_in_progress` here as these states can't exist after the submission deadline.

Comment (jaleszek) [inline app/views/content_only/post_submission/_submitted.html.slim]: in 24hours after the submission deadline so in fact for sure after the shortlisting

**Pull request merged**